apps/website/next.config.js

```diff
@@ -70,6 +70,11 @@ const config = {
         protocol: "https",
         hostname: "alveus.nyc3.cdn.digitaloceanspaces.com",
       },
+      // Local Dev
+      {
+        protocol: "http",
+        hostname: "localhost",
+      },
```

Comment:
Should this be its own PR to document the change?

Reply:
I can do that if you want.